subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c

```diff
@@ -509,7 +541,11 @@ psa_status_t cracen_platform_keys_provision(const psa_key_attributes_t *attribut
 		return PSA_ERROR_INVALID_ARGUMENT;
 	}
 
-	psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
+	psa_status_t status =
+		verify_access(MBEDTLS_SVC_KEY_ID_GET_OWNER_ID(psa_get_key_id(attributes)), key_id);
```

Comment:
For consistency lets have MBEDTLS_SVC_KEY_ID_GET_KEY_ID(key_id) instead of having the key_id as is. You do that in line 511, so lets stay consistent.